front/components/sparkle/navigation.tsx

```diff
@@ -115,41 +120,39 @@ export const topNavigation = ({
 };
 
 export const subNavigationConversations = ({
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   owner,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   current,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   subMenuLabel,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   subMenu,
 }: {
   owner: WorkspaceType;
-  current: SubNavigationAdminId;
+  current: SubNavigationConversationsId;
   subMenuLabel?: string;
   subMenu?: SparkleAppLayoutNavigation[];
 }) => {
   const nav: SidebarNavigation[] = [];
 
   // To be added for personal assistants view.
 
-  // nav.push({
-  //   id: "assistants",
-  //   label: null,
-  //   variant: "secondary",
-  //   menus: [
-  //     {
-  //       id: "personal_assistants",
-  //       label: "My Assistants",
-  //       icon: RobotIcon,
-  //       href: `/w/${owner.sId}/builder/assistants`,
-  //       current: current === "personal_assistants",
-  //       subMenuLabel:
-  //         current === "personal_assistants" ? subMenuLabel : undefined,
-  //       subMenu: current === "personal_assistants" ? subMenu : undefined,
-  //     },
-  //   ],
-  // });
+  if (owner.role === "user") {
+    nav.push({
+      id: "assistants",
+      label: null,
+      variant: "secondary",
+      menus: [
+        {
+          id: "personal_assistants",
+          label: "My Assistants",
```

Comment:
note: it was "Assistants" in the Figma. Personally fine with My Assistants too, just checking it's not an overlook

Reply:
There is both in the Figma :)